### ISSUE TITLE
Idea thread

### ISSUE THREAD

**Issue by Quantum64** [Quantum64/ExGregilo#20]
Now that most of the tickets have been resolved, it's time for new content or features, but I'm not sure what to add. Any ideas?

**Comment by Hakameda** [comment]
Some method of turning crushed ores into Ore blocks without duping byproducts. like 3:1 Crushed Ores to Ores

**Comment by Quantum64** [comment]
I added a greg equivalent of the compressed hammer and auto compressed hammer today. May also add a greg auto heavy sieve and advanced heavy sieve.

**Comment by richardhendricks** [comment]
Use sticky resin + crushed ores to turn it into an ore block.  That would be sufficiently funny.  sticky resin is easy to get from rubber trees, bees, etc.

**Comment by richardhendricks** [comment]
Does the heavy sieve have a cute little steve inside working it?

https://www.curseforge.com/minecraft/mc-mods/ex-compressum

It would be fun to use a zombie brain on it and animate.

**Comment by Hakameda** [comment]
Lmao, you could add a use for refined glue, literally glue 3-4 crushed ores
together

**Comment by Hakameda** [comment]
How hard would it be to add the world gen stuff, similar to how garden of
glass works?

**Comment by Quantum64** [comment]
Not sure, haven't looked into worldgen with forge before.

I'm currently working on a multiblock variant of the auto sieve that will be more efficient than higher tiers of normal auto sieves and also work with compressed blocks.

**Comment by Quantum64** [comment]
I have exams starting tomorrow and lasting the rest of the week that I should probably study for. There might not be updates for some time.

**Comment by WSpadaro** [comment]
I think that the dust outputs from the Auto Sieve might be a little bit rough. In my world I just set up a creative barrel of dusts and an VII machine + ultimate battery and have been letting it just run. I still only have 2 pieces of tiny chrome. I don't see how HV tier will be possible. 

Also, I'm not sure how to get Tungsten from Ex-Gregilo. 

And the last thing is, I don't see a method to get gallium that doesn't require HV but gallium is required for gallium arsenide in getting diodes to progress.

**Comment by richardhendricks** [comment]
Chrome is also obtainable from centrifuging redstone. You'll need large quantities, the easy thing to do is get the redlon from thaumcraft.  Which does raise a point, how the heck do people get nodes and TC flowers?

For gallium, it is reasonably easy via bees, but otherwise comes from Sphalerite in the nether.  So maybe add it as a byproduct from netherrack?

The other item is arsenic.  It should not be as easy to get gallium IMO since it is normally only available via small ores in the Nether or from cobaltite found in nickel veins in the Twilight Forest.  I suggest getting it from soul sand, since that would require locating a source of the sand and collecting it.

**Comment by WSpadaro** [comment]
AFAIK Auto Sieves do not produce redstone. Also, I do not think this mod should rely on Forestry because Gregtech doesn't.

**Comment by Quantum64** [comment]
I might have broken redstone output, I'll fix that at some point.

**Comment by Hakameda** [comment]
@wspadaro

**Comment by Hakameda** [comment]
Derp forgot to add a comment. Do you plan on making a just gregtech skyblock?

**Comment by claws61821** [comment]
@Hakameda I'm going to chime in on this with my own opinion.

You're making a mod, not a pack, and that's all well and good. Your mod inherently requires another mod, Gregtech, because that's your mod's theme; again fine. If you decide to have it require additional mods because they're needed to support items, blocks, or machines which you're adding, that still makes sense.

What doesn't make sense is when you, or any mod developer, require mods compatible with but unrelated to your core theme (in this case any beyond Gregtech and those specifically necessary for Skyblock) because you've decided to make standard progression of the mods you're basing your own upon dependent upon unrelated mods which they did not depend upon before (e.g. requiring TC or Forestry to progress Gregtech). Certainly you can provide configuration settings which take advantage of mod compatibility in that fashion, but making an unrelated mod a mandatory prerequisite in that manner, as described above, tends to alienate players who are uninterested in that mod but enjoy the standard or altered-standard progression of vanilla or of the mod upon which yours is based.

tl;dr:
Stick to GT base requirements for standard progression and only alter progression requirements if compatible mods are detected or if changes are made to configuration files.

**Comment by Hakameda** [comment]
I'm not making a mod, I'm modifying stuff to work with a modpack that already exists. So my suggestions are bug fixes for the base ex gregilo and stuff that would help the mod itself. Then simply making it work for what I need. The above question was because in the progress of making it work for gtnh, I'm making sure all progression exists and in doing that I can also make sure it exists for the base now too. As some stuff is missing as per discussion about it

**Comment by claws61821** [comment]
Sorry, @Hakameda 
I got you mixed up with Quantum earlier, and thought your "just GTSB" comment held a different context than it does above.
@Quantum64 Will the MB sieve work with multiple tiers of compression?

**Comment by Hakameda** [comment]
What quantum is doing is also good, better alternative to ex compressum. I love gt tool style. But mostly he made it so gregtech progression is possible as if it was just gregtech, it just doesn't use the new circuit crap

**Comment by Quantum64** [comment]
@claws61821 MB sieve will definitely support compressed blocks, and I'm thinking of adding oredict compatible compressed dust/sand/gravel blocks so this mod doesn't have a progression dependency on ExCompressum or ExtraUtils. I'm making the power scaling on auto sieve tiers more aggressive so that the MB sieve will make sense for anything above HV, even for non compressed blocks.

**Comment by WSpadaro** [comment]
@Quantum64 What's the point of making higher tiers of Sieves less effective. The advantage of the multi block should be that it's cheaper to make a multiblock than 6 HV sieves or whatever imo. One should be faster or more efficient or cheaper than the other. Not all three. No point in having something that is strictly worse because then you mine as well not include it, no?

**Comment by Hakameda** [comment]
Do the sieves support processing arrays if they are gregtech machines?

**Comment by Quantum64** [comment]
@WSpadaro I don't plan on making the MB version much (if at all) faster than the normal versions. It will be more efficient and support compressed blocks. The auto sieve power usage is in my opinion too low anyway, it's on par with the electric furnace right now.

**Comment by claws61821** [comment]
Oredict-compatible compressed blocks sounds like a good plan. Would that be a similar utility to AOBD from 1.6.4 and 1.7.10; essentially auto-generating a block for every detected ore and etc? If you're including multiple tiers of compression, how many do you plan to include, or to at least include compatibility for? I.E. will Ex-Gregilo sieves be compatible with e.g. 8x compressed sand, or will they stop at e.g. 2x compressed sand?

**Comment by claws61821** [comment]
Idea for new machines (may already be implemented; I really need to play with this mod soon):

Material Recovery Air Filters: Tiered industrial machines using specific filter screens and energy/tick to 'recover' particulate contaminants (i.e. dusts) from the air. Essentially, a [u]low-production slow[/u] source of 'infinite' random Small Dust Piles requiring only a filter and a source of energy. Recommended tiers: Manual, LV, MV, HV. Filter screens should have a durability value and function similar to sieve meshes.

Material Recovery Water Filters: Similar to the above Air Filters, these would instead recover dusts and other materials from water, in exchange for negating or diminishing the energy requirement. Recommended tiers: Passive (manual), LV, MV (maybe). If the Cauldron and the Ore Washing Plant can be modified to retain or output 'Dirty Water' instead of vanishing used Water, this could be sent through the Water Filter to produce a higher quantity of dust than regular Water at the expense of higher durability damage to the screen. In addition to the dusts, the Water Filter could also output a diminished volume of water from that present at input (optional, might be too easy to abuse).

Just a couple of ideas I had from looking at the Flint and String Mesh thread earlier. I don't expect this to get added but if it does then I think it could be interesting.

**Comment by draknyte1** [comment]
> Do the sieves support processing arrays if they are gregtech machines?

No.
Processing array support is hard coded into Gregtech and there is no support for addon machines.

As a fellow addon creator, the less your mod relies on anything (Even in a dev enviro) the better.
In reality, your mod shouldn't require anything outside GT and GT functions to support every other mod.

Listening to the GTNH players for content ideas is something to listen to with a grain of salt, because there are significant changes with their fork of GT and they've got certain Ores Tiered to Certain dimensions (Hence the TF comment by Richard above) and they forget this more often than not. While you are working with them to enable a GTNH skyblock, just keep in mind that's it's a secondary feature of support but your mod needs to function 100% with plain old GT.

**Comment by Quantum64** [comment]
My mod technically only has soft dependencies and I plan for all progression to be possible with just GT eventually (phasing out the requirement for early game ExNihilo sieves at some point).

**Comment by Hakameda** [comment]
That's why the suggestion for Flint/string. The hope was to not use ex
nihilo sieves

On Dec 20, 2017 10:13 AM, "Dylan" <notifications@github.com> wrote:

My mod technically only has soft dependencies and I plan for all
progression to be possible with just GT eventually (phasing out the
requirement for early game ExNihilo sieves at some point).

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/Quantum64/ExGregilo/issues/20#issuecomment-353124820>,
or mute the thread

**Comment by Quantum64** [comment]
From now on I will only be accepting PRs that pass the TravisCI build test. If your PR doesn't pass, look at the recorded error and fix the issue.

**Comment by Hakameda** [comment]
With work being done on gt5u for 1.10, Will this also get ported at some point?

**Comment by Quantum64** [comment]
Actually I am working on GT-CE (formerly GT5U-10) now! Once it's ready I will be porting this mod.

**Comment by richardhendricks** [comment]
How about having the multiblock sieve work like the Railcraft steam furnace, where it has a 3x3 input and output grid, and does 9 items at a time?  Having a multiblock machine that does things faster should be easier on server load since players won't need to expand horizontally, they can expand technologically.

**Comment by Hakameda** [comment]
Any chance of bug fixes for 1.7.10 or still pretty busy?

**Comment by Quantum64** [comment]
Not feeling too motivated to work on Minecraft mods currently. Mostly because of the massive changes that will need to be made to CE with the 1.13 update, so I'm sort of trying to wait that out.

**Comment by Hakameda** [comment]
No worries, another dude is helping a bit. Only issue is the railcraft jar
shit

**Comment by draknyte1** [comment]
What's railcraft jar issue?

**Comment by Hakameda** [comment]
In the build script, build.gradle. It can't download railcraft 9.4.0.0
which just errors out
